xc/xenops_server_xen.ml

```diff
@@ -1000,10 +1015,15 @@ module HOST = struct
         features_hvm.(3) <- Int64.logor features_hvm.(3) 0x2L ;
         (* Set X86_FEATURE_IBS in e1c for HVM guests *)
         features_hvm.(3) <- Int64.logor features_hvm.(3) 0x400L ;
```

Comment:
This (and further above) is also CPUID debt wanting to move out of xenopsd.

Reply:
dropped